src/Implementations/Composer.php

```diff
@@ -27,18 +26,15 @@ class Composer implements ComposerInterface
 
     /**
      * @param JFI $file
-     * @param TCDI $decider
+     * @param CPI $processor
      * @throws InvalidArgumentException if file is unusable
```

Comment:
exception isn't been thrown anymore. or maybe in JFI?

Reply:
it's (possibly) thrown in JFI, yes